### PR TITLE
Add Closed Areas to Zone and District Sheets

### PR DESCRIPTION
**Pull request by ElderLoForte**
adding all of Weewoo to main

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (HarrierPigeon) [inline .vscode/settings.json:1]: You can probably delete this file.

**Review by HarrierPigeon — COMMENTED** [review]
Need to delete that settings.json file, everything else looks good!

**Review by ElderLoForte — COMMENTED** [review]
should have fixed that now

**Review by HarrierPigeon — APPROVED** [review]
Looks awesome!

**Pull request merged**